### ISSUE TITLE
ArgumentNullException in CSharpConvertToAsyncMethodCodeFixProvider

### ISSUE THREAD

**Issue by KirillOsenkov** [dotnet/roslyn#6477]
```
using System.Threading.Tasks;

class C
{
    static async void Main()
    {
        try
        {
            await 
            await Task.Delay(100);
        }
        finally
        {
        }
    }
}
```

Hover over the first await.

System.ArgumentNullException: Value cannot be null.
Parameter name: node
   at Microsoft.CodeAnalysis.CSharp.CSharpSemanticModel.GetSymbolInfoFromNode(SyntaxNode node, CancellationToken cancellationToken) in F:\Builds\6272\Roslyn\Main-Signed-Release\src\Open\src\Compilers\CSharp\Portable\Compilation\CSharpSemanticModel.cs:line 4228
   at Microsoft.CodeAnalysis.CSharp.CSharpSemanticModel.GetSymbolInfoCore(SyntaxNode node, CancellationToken cancellationToken) in F:\Builds\6272\Roslyn\Main-Signed-Release\src\Open\src\Compilers\CSharp\Portable\Compilation\CSharpSemanticModel.cs:line 4378
   at Microsoft.CodeAnalysis.CSharp.CodeFixes.Async.CSharpConvertToAsyncMethodCodeFixProvider.<GetMethodDeclaration>d__5.MoveNext() in F:\Builds\6272\Roslyn\Main-Signed-Release\src\Open\src\Features\CSharp\CodeFixes\Async\CSharpConvertToAsyncMethodCodeFixProvider.cs:line 62

**Comment by Pilchie** [comment]
@jmarolf Isn't this a dupe?

**Comment by jmarolf** [comment]
This does not repro on the latest stabilization build

**Comment by jmarolf** [comment]
fixed by #6488
